WordPress/WordPressUITests/WPUITestCredentials.swift

```diff
@@ -3,7 +3,8 @@ struct WPUITestCredentials {
     static let testWPcomUserEmail: String = "e2eflowtestingmobile@example.com"
     static let testWPcomUsername: String = "e2eflowtestingmobile"
     static let testWPcomPassword: String = "mocked_password"
-    static let testWPcomSiteAddress: String =  "e2eflowtestingmobile.wordpress.com"
+    static let testWPcomSiteAddress: String =  "infocusphotographers.wordpress.com"
+    static let testWPcomSiteDisplayName: String =  "infocusphotographers.com"
```

Comment:
I've been thinking about this, and what do you think of changing this variable to something like `testWPcomSitePrimaryAddress`? I'm finding the term `DisplayName` a little confusing as I look through the rest of the code because I keep thinking it means "Site Title" whereas "Primary Address" is a term used on wp.com to refer to the address that's used/displayed everywhere.

Reply:
If that's the more correct term I'm definitely in favour of it! Changed!